scripts/measures/get-inverse-measures-from-pdfs.js

```diff
@@ -0,0 +1,59 @@
+// running this will print out whether each measure is an inverse measure
+// claims pdfs can be downloaded from https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip
+// unzip and give the path to the Claims-Quality-Measures
+// usage ./get-inverse-measures-from-pdfs.js <path to folder of claims quality measures pdfs>
+
+const fs = require('fs');
+const program = require('commander');
+const pdfToText = require('pdf-to-text');
+const p = require('path');
+
+let folderPath = null;
+
+function setPath(path) {
+  folderPath = path;
+}
+
+program
+  .version('0.0.1')
+  .arguments('<path>')
+  .action(setPath)
+  .parse(process.argv);
+
+if (!folderPath) {
+  console.log('Missing required argument <path>');
+  process.exit(1);
+}
+
+function getIsInverse(path, cb) {
+  pdfToText.pdfToText(path, function(err, data) {
+    if (err) {
+      cb(false);
+      return;
+    }
+
+    let found = data.match(/inverse measure/i); // ignore case
```

Comment:
did you do a comparison between inverse measure and INVERSE MEASURE to see if anything is picked up? I didn't see any pdfs with lower case on a spot check, but am curious to see if there are any that are lower case.

Reply:
I just ran it with lower case for you and the result was the same.